app/test_main.py

```diff
@@ -1,3 +1,28 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age,dog_age,result",
+    [
+        (14, 14, [0, 0]),
+        (28, 28, [3, 2]),
+        (100, 100, [21, 17]),
+        (-50, -45, [0, 0]),
```

Comment:
The test case with negative values `(-50, -45, [0, 0])` assumes that negative ages should return `[0, 0]`. Ensure that the `get_human_age` function is designed to handle negative inputs appropriately. If the function should raise an error for negative inputs, the test should reflect that behavior.